tests/unit/views/pacbio/PacbioRunShow.spec.js

```diff
@@ -191,6 +192,27 @@ describe('Run.vue', () => {
     })
   })
 
+  describe('#created', () => {
+    it('"findPools" gets called with a list of barcodes', () => {
```

Comment:
It might make sense to skip this with a reason so we know it is there. As it stands the test will pass.